Tasgrid/tasgridWrapper.hpp

```diff
@@ -56,6 +56,7 @@ enum TypeCommand{
 
     command_evaluate,
     command_integrate,
+    command_differentiate,
```

Comment:
Good, but you should also add the derivative weights, e.g., similar to the get quadrature weights and get interpolation weights in matlab.

Reply:
Done. 

Note that the interpretation of these weights can be a bit tricky (see the documentation in the new file `InterfaceMATLAB/tsgGetDifferentiationWeights.m`).